gh-badges/templates/default-template.json

```diff
@@ -1,4 +1,7 @@
 {
   "name": {{=JSON.stringify(it.text[0])}},
-  "value": {{=JSON.stringify(it.text[1])}}
+  "label": {{=JSON.stringify(it.text[0])}},
+  "value": {{=JSON.stringify(it.text[1])}},
+  "message": {{=JSON.stringify(it.text[1])}},
+  "color": {{=JSON.stringify(it.color || null)}}
```

Comment:
Could you transfer over the other properties from _shields_test-template? `logoWidth` and `labelColor`

The reasoning is that I’d like the output of the JSON badge to as closely as possible match the input to the `makeBadge` function. It should make the server logic and the badge generation code testable in complete isolation. It also makes the JSON endpoint more flexible because it contains all the data for the badge, you can hit that endpoint (on e.g. the live server), manipulate it the response, and then pipe it to your own copy of `makeBadge`. Shields has a ton of layers – though good on us, we keep peeling them away! I think the more we can make these schemas consistent, the lower the mental overhead will be.